### ISSUE TITLE
invalid plugin key URI "cri" expect io.containerd.x.v

### ISSUE THREAD

**Issue by juniway** [kata-containers/kata-containers#4124]
### Maybe it's time to update your howto on config on containerd's config.toml

If I apply your config syntax (according to https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/containerd-kata.md)to containerd's `config.toml`, contained won't be able to start, and it will complain about the error:

```
containerd: failed to load TOML from /etc/containerd/config.toml: invalid plugin key URI "cri" expect io.containerd.x.vx
```

![image](https://user-images.githubusercontent.com/3980841/164366286-48a6aa7c-7cb6-492c-8692-69d8693736b4.png)

The config in your **howto page** is:
```
[plugins.cri.containerd.runtimes.kata]
         runtime_type = "io.containerd.kata.v2"
	 privileged_without_host_devices = true
	 [plugins.cri.containerd.runtimes.kata.options]
	   ConfigPath = "/etc/kata-containers/config.toml"
```
Basically, `cri` should be modified to `io.containerd.grpc.v1.cri` (If you are using contained version 1.5.10)

After modification, above config becomes:

```
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
         runtime_type = "io.containerd.kata.v2"
	 privileged_without_host_devices = true
	 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata.options]
	   ConfigPath = "/etc/kata-containers/config.toml"
```

**Comment by liubin** [comment]
Hi @juniway can you share your containerd's config.toml and your containerd's version?
